pages/rewards/staking_rewards.md

```diff
@@ -0,0 +1,30 @@
+# Staking Rewards
```

Comment:
Awesome documentation! Should we mention validator commission (which is under the staking rewards concept)? Validators have had [questions](https://dydx-team.slack.com/archives/C05TJ98ATC2/p1701790240437369) about this as well.

Reply:
We have a few places in this page and the `overview` page that mentions `validator commission`. Do you think it's sufficient at the moment? Or do you think it's better to add it in the next paragraph here?